collector/index.mdx

```diff
@@ -29,3 +29,33 @@ The collector makes only outbound connections, it does not listen on any ports.
 Heroku-based setup, where the collector listens on the dyno's PORT for Heroku to consider the application running,
 as well as to receive database logs by serving as a Heroku log drain.
 
+## Data collection and Snapshot submission Frequency
+
+The pganalyze collector gathers metrics and metadata through various snapshots, each with specific cadences.
+This section explains the types of data collected and when different snapshot types are sent.
+
+There are three different types of snapshots that are sent to the pganalyze app.
+
+* **Compact Activity Snapshot**: Sent every 10 seconds
+* **Compact Logs Snapshot**:
+  * Streaming: Sent approximately every 10 seconds
+  * Download: Sent every 30 seconds
+* **Full Snapshot**: Sent every 10 minutes (logged as just "snapshot")
+
+Successful submissions are logged as follows:
+
+```
+Submitted compact activity snapshot successfully
+Submitted compact logs snapshot successfully
+Submitted snapshot successfully
+```
+
+These snapshots include metrics and metadata collected according to specific schedules and serve the following purposes:
+
+| Data             | Collection Interval | Snapshot Type    | Features                    |
+|------------------|---------------------|------------------|-----------------------------|
+| Connections      | Every 10 seconds    | Compact Activity | Connections                 |
```

Comment:
To save horizontal space, I think we can drop "every" from the collection intervals.

Reply:
Yeah we could drop indeed. Jens and I talked about this a bit, here is how it looks like:

![image](https://github.com/pganalyze/pganalyze-docs/assets/911433/90d85fa5-4e0e-4355-9d2a-8c7b3e87781b)

Given that dropping "Every" won't help much re: horizontal space, we are leaning towards to keep "Every" for every thing, to make this part clear.